start-docker-compose.sh

```diff
@@ -31,11 +34,16 @@ while getopts ':dhz' OPTION; do
       Help
       exit;;
     z)
-      # Do a pull first to make sure we use the latest ZAC Docker Image
-      echo "Pulling latest ZAC Docker Image.."
+      echo "Pulling latest ZAC Docker Image ..."
       startZac=true
       docker compose pull zac
       ;;
+    l)
+      echo "Building ZAC Docker Image ..."
+      startZac=true
+      export ZAC_DOCKER_IMAGE=infonl/zaakafhandelcomponent:latest
+      docker build . -t $ZAC_DOCKER_IMAGE
```

Comment:
Better to use our `buildDockerImage` Gradle task.

Reply:
Done